lib/experimental/new_embed.dart

```diff
@@ -177,16 +187,22 @@ class NewEmbed {
     }
 
     tabController.setTabVisibility('test', false);
-    showTestCodeCheckbox = DInput(querySelector('#show-test-checkbox'));
-    showTestCodeCheckbox.onClick.listen((e) {
-      _showTestCode = !_showTestCode;
-      tabController.setTabVisibility('test', _showTestCode);
-    });
+    showTestCodeCheckmark = DElement(querySelector('#show-test-checkmark'));
+    showTestCodeMenuItem = DElement(querySelector('#show-test-menu-item'));
 
     morePopover = DElement(querySelector('#more-popover'));
     menuButton = DisableableButton(querySelector('#menu-button'), () {
-      var popoverHidden = morePopover.hasAttr('hidden');
-      morePopover.toggleAttr('hidden', !popoverHidden);
+      menu.open = !menu.open;
+    });
+    menu = MDCMenu(querySelector('#main-menu'))
```

Comment:
This is fine for the moment, but if we get more items in this menu, we'll probably want to break it out into a class of its own.